index.js

```diff
@@ -981,7 +981,7 @@
   //. > S.Nothing().sequence(S.Either.of)
   //. Right(Nothing())
   //.
-  //. > S.Just(Right(42)).sequence(S.Either.of)
+  //. > S.Just(S.Right(42)).sequence(S.Either.of)
```

Comment:
Doctests have access to variables in scope, but this is not the case when evaluating these expressions in isolation via [`vm.runInNewContext`][1].


[1]: https://nodejs.org/api/vm.html#vm_vm_runinnewcontext_code_sandbox_options